open-apparel-registry/open-apparel-registry
# Upgrade terraform to 1.1.9

## Overview
A terraform upgrade was necessary for a desired feature surfaced from work in another PR in the parent issue. 

Connects #1998 

## Notes
This was a multi-step upgrade that involved jumping through multiple versions to minimize the amount of code changes necessary at each step. Due to the nature of the upgrade, the terraform code was applied at each step, so code covered by this PR is currently in place on the OSH staging environment.

## Testing Instructions
This code is currently deployed to staging, so testing should be done as a verification of the changes.

* ensure that jenkins completed successfully
* review the console output in jenkins to verify that nothing was changed that affects the functioning of the site
* navigate the staging site to ensure correct functionality
* submit a list to ensure that submitted lists are processing correctly.

## Checklist
